index.Rmd

```diff
@@ -1,105 +1,299 @@
+
+I loaded required packages
+
+```{r, message=F, warning=F}
+library(threejs)
+```
+
+##Results2
+I firstly prepared and loaded the data and then with the package called "streejs" I plot the exact locations of landing and mining sites on the Martian global map. I created a cvs file where I stored the latitudes and logitude of the future landing and mining sites selested and call that file in R.
+
+
+```{r}
+library(threejs)
+library(csvread)
+the_martian <- read.csv("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landingMining_sites/the_martian.csv", comment.char="#")
+                        
+#the_martian.csv <- ("C:/Users/Owner/Desktop/sare  backup/c/user #hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landi#ngMining_sites/the_martian.cvs")
+  
+
+globejs(img = "C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landingMining_sites/Mars_Viking_MDIM21_ClrMosaic_global_1024.jpg", bodycolor="#555555", emissive="#444444",
+        lightcolor="#555555",
+        lat = the_martian$x,
+        long = the_martian$y)
+#        lat="C:/Users/Owner/Desktop/sare  backup/c/user #hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landi#ngMining_sites/the_martian.csv/the_martian$x", #The latitude when I use this code is not showing up and I need to work on it little #bit.
+
+ #       long = "C:/Users/Owner/Desktop/sare  backup/c/user #hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/visualize_of_landi#ngMining_sites/the_martian.csv/the_martian$y") #The longitude when I use this code is also still not showing up and I need to work on it #little bit.
+      
+```
+
+
+
 # Conclusions
 
-[~200 words]
+From all sites I selected as training sites I choosed 9 sites as training sites (Si polar swath"I00831006"; Lyot crater"V00823007"; Cerberus fossae"V00825005";Echus chasma"I00839002"; Sinus Meridiana west"I00849005"; Syrtis Major"I00853002"; Ophir planum"I00864002"; Marte vallis"I00868006; Eastern Meridiani"I00836002"), Meridiana west seems to be the best one. The band ratio combination is used to determine the presence, abundance and distribution of water, water ice, ice or volatiles materials in each site before selecting some specific spots within Meridiana west as future landing/mining site.This is done using R language meaning that R is a suitable tool offering the user the choice to manipulate the data in the way he or she wants as well as to perfom diverses task on Earth and in space. R is definitely a great too that can help explorers, planetary scientists to overcome some difficulties they could not handle with other softwares.
```

Comment:
How do you define the "best" location?  There is no connection between the data and the sites you selected.